site2/docs/concepts-messaging.md

```diff
@@ -44,6 +44,7 @@ Messages published by producers can be compressed during transportation in order
 
 * [LZ4](https://github.com/lz4/lz4)
 * [ZLIB](https://zlib.net/)
+* [ZSTD](https://facebook.github.io/zstd/)
```

Comment:
Please fix text above:
`supports two types of compression:`
to
`supports the following types of compression:`

Reply:
Good catch! Please open directly a pull request to fix it 😀

Reply:
ready https://github.com/apache/pulsar/pull/3789